docs/editors-guide/externally-managed-content.md

```diff
@@ -0,0 +1,33 @@
+## Automatic Mondo Quality Control Checks for Externally Managed Content
+A subset of the extensive Mondo QC checks are automatically run on externally managed content. Any content from external providers that does not pass these QC checks is not included in Mondo until the content is updated and passes the QC checks. Reports of these QC checks will be available for review of any errors so that any needed changes can be made.
+
+### QC Checks for Externally Managed Content
+| Error | Description | SPARQL Query | 
+|:------|:------------|:-------------|
+| duplicate_scoped_synonym | An entity has duplicate synonyms with different properties (e.g. the same broad and related synonym). This causes ambiguity. | <a href='https://robot.obolibrary.org/report_queries/duplicate_scoped_synonym' target='_blank'>duplicate scoped synonym</a> |
+| invalid_xref | A database_cross_reference is not in CURIE format. | <a href='https://robot.obolibrary.org/report_queries/invalid_xref' target='_blank'>invalid xref</a> |
```

Comment:
Suggestion: 
A database_cross_reference in the EMC file is not in CURIE format.

Reply:
updated